endtoendtests/Azure.Functions.Java.Tests.E2E/Azure.Functions.Java.Tests.E2E/Constants.cs

```diff
@@ -20,9 +20,9 @@ public static class Constants
         public static string TestQueueMessage = "Hello, World";
 
         //Blob tests
-        public static string TriggerInputBindingBlobContainer = "test-triggerinput-java";
-        public static string InputBindingBlobContainer = "test-input-java";
-        public static string OutputBindingBlobContainer = "test-output-java";
+        public static string TriggerInputBindingBlobContainer = "test-triggerinput-java-new";
```

Comment:
We can create an issue to fix this. (I mean update the pipeline or clean up code.)

Reply:
Yes great point looks like the createBlob and cleanBlob for a reason not working in the e2e test.